tests/baselines/reference/numericClassMembers1.js

```diff
@@ -19,21 +19,21 @@ class C236 {
 var C234 = (function () {
     function C234() {
         this[0] = 1;
-        this[0.0] = 2;
+        this[0] = 2;

```

Comment:
This is an error, it should be using the raw literal text. I have a similar issue with binary/octal integers I need to investigate.